engine/classes/Elgg/Application/CacheHandler.php

```diff
@@ -29,6 +29,7 @@ class CacheHandler {
 		'js' => 'application/javascript',
 		'json' => 'application/json',
 		'map' => 'application/json',
+		'mjs' => 'application/javascript',
```

Comment:
should be mimetype `text/javascript` https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types